lib/league_stats.rb

```diff
@@ -9,13 +9,13 @@ def initialize(games, teams)
 
   def highest_offense
     @teams.max_by do |team|
-      team.calculate_offense
+      team.average_goals(team.total)
     end.team_name
   end
 
   def lowest_offense
     @teams.min_by do |team|
-      team.calculate_offense
+      team.average_goals(team.total)
```

Comment:
I believe these should give the same answer. Average goals scored vs. total goals scored.